idseq_pipeline/commands/common.py

```diff
@@ -71,23 +71,19 @@ def save_to_s3(self):
             self._save()
             execute_command("aws s3 cp --quiet %s %s/" % (self.stats_path, self.s3_output_dir))
 
-    def get_total_reads(self):
-        for item in self.data:
-            if "total_reads" in item:
-                return item["total_reads"]
-        # check run star if total reads not available
+    def get_item(self, function_name):
```

Comment:
get_total_reads is no longer so special;   if we have a predigested post-filtered input, that came with a "total_reads";  otherwise we initialize "total_reads" at the start of host filtering;   so it's always there, and these old gymnastics are unnecessary